core/upgrade/upgrades.php

```diff
@@ -860,6 +860,10 @@ public static function _v_3_0_0_move_default_typography_to_kit( $updater ) {
 		return self::move_settings_to_kit( $callback, $updater );
 	}
 
+	public static function plugin_installed_before_v_3_0_0() {
+		return Plugin::$instance->get_install_time() < strtotime( '23 August 2020' );
```

Comment:
Don't work with time, add an upgrade method to adds a `wp_option` for accurate indication.

Reply:
This is a common approach across our platform